src/connectedk8s/HISTORY.rst

```diff
@@ -2,6 +2,13 @@
 
 Release History
 
+1.3.18
+++++++
+
+* Checking Outbound connectivity to OBO endpoint 
+* Cleaning up CRD in case release namespace is delete/not-present
```

Comment:
cleaning up stale CRDs if present during onboarding (even in absence of azure-arc release)